src/layouts/core-dev-dot-layout/core-dev-dot-layout.module.css

```diff
@@ -30,13 +30,13 @@
   --tablet-width-breakpoint: 1000px;
 
   /* Font overwrites */
-  --font-body: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
-    Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+  --font-body: system-ui, -apple-system, blinkmacsystemfont, 'Segoe UI', roboto,
+    helvetica, arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
     'Segoe UI Symbol';
-  --font-display: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',
-    Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+  --font-display: system-ui, -apple-system, blinkmacsystemfont, 'Segoe UI',
+    roboto, helvetica, arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
     'Segoe UI Symbol';
-  --font-monospace: ui-monospace, Menlo, Consolas, Ubuntu Mono, monospace;
+  --font-monospace: ui-monospace, menlo, consolas, ubuntu mono, monospace;
```

Comment:
A note, outside of the scope of this PR: I think we want to use [the HDS `font-stack` tokens](https://github.com/hashicorp/design-system/blob/main/packages/tokens/dist/devdot/css/tokens.css#L241-L243) here, but there are some differences between the tokens and what we have here, so I think it's important we double check with Design and Design Systems folks about this.

New ticket: https://app.asana.com/0/1201010428539925/1201980530846410/f